tests/syntax/test_operators.py

```diff
@@ -503,6 +503,56 @@ def test_object_in_region_3d():
     assert p == (True, True, False, False)
 
 
+# Intersects
+def test_intersects_obj_obj():
+    p = sampleParamPFrom(
+        """
+    obj1 = new Object at (-1,0,0.1), with allowCollisions True
+    obj2 = new Object at (1,0,0), with allowCollisions True
+    obj3 = new Object with width 10, with length 10, with height 10, with allowCollisions True
```

Comment:
Could you please add a case where neither the center nor the corners of the objects lie in each other, but they still intersect?